HintList.py

```diff
@@ -557,17 +557,17 @@ def tokens_required_by_settings(world: World) -> int:
     'HF GS Cow Grotto':                                            ("a #spider behind webs# in a grotto holds", None, ['overworld', 'sometimes']),
     'HF Cow Grotto Cow':                                           ("the #cobwebbed cow# gifts", "a #cow behind webs# in a grotto gifts", ['overworld', 'sometimes']),
     'ZF GS Hidden Cave':                                           ("a spider high #above the icy waters# holds", None, ['overworld', 'sometimes']),
-    'Wasteland Chest':                                             (["#deep in the wasteland# is", "beneath #the sands#, flames reveal"], None, ['overworld', 'sometimes']),
+    'Wasteland Chest':                                             (["#deep in the wasteland# is", "beneath #the sands#, flames reveal"], "a chest spawned by #lighting flames in Haunted Wasteland# contains", ['overworld', 'sometimes']),
```

Comment:
To keep it consistent with other hints, it should say "**the** Haunted Wasteland"